docs/mkcloud.md

```diff
@@ -20,6 +20,14 @@ or testing purposes.
   $ sudo service libvirtd start  # to start the daemon
   ```
 
+  If you are using `libvirtd >= 1.3.0` then you should also start the
+  `virtlogd` service.
+
+  ```
+  $ sudo service virtlogd status # to check the status
+  $ sudo service virtlogd start  # to start the daemon
```

Comment:
@hwoarang The `Also=` part enables the socket part of `virtlogd` but not the service part. For that you still have to `systemctl start virtlogd.service`.